setup.py

```diff
@@ -0,0 +1,66 @@
+"""nprintml"""
+import pathlib
+
+from setuptools import find_packages, setup
+
+
+README_PATH = pathlib.Path(__file__).parent / 'README.md'
+
+INSTALL_REQUIRES = [
+    'argparse-formatter==1.2',
+]
+
+_DEV_REQUIRES = [
+    'argcmdr==0.7.0',
+    'bumpversion==0.6.0',
+    'twine==3.2.0',
+    'wheel==0.35.1',
+]
+
+_TESTS_REQUIRE = [
+    'tox==3.20.1',
+]
+
+EXTRAS_REQUIRE = {
+    'dev': _DEV_REQUIRES + _TESTS_REQUIRE,
+    'test': _TESTS_REQUIRE,
+}
+
+
+setup(name='nprintml',
+      version='0.0.0',
+      description='DESCRIPTION GOES HERE',
+      long_description=README_PATH.read_text(),
+      long_description_content_type="text/markdown",
+      url='https://github.com/nprint/nprintml',
+      # license='xxx',  # FIXME
```

Comment:
This is intended to specify a license classifier. So I guess –

```suggestion
      # license='License :: OSI Approved :: Apache Software License',
```